### PR TITLE
Work3

### PR DESCRIPTION
**Pull request by Irina996**

### CODE REVIEW

File: ReviewSite/header/__button/header__button.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.header__button {
+    background-color: #1492AE;
+    color: black;
+    margin-top: 30px;
+    float: right;
```
Comment (katetushkan) [inline ReviewSite/header/__button/header__button.css]: no floats.

File: ReviewSite/header/__button/header__button.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    border-width: 0;
+    font-size: medium;
```
Comment (katetushkan) [inline ReviewSite/header/__button/header__button.css]: use strict values.

File: ReviewSite/header/header.css [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.header {
+    color: black;
+    height: 70px;
```
Comment (katetushkan) [inline ReviewSite/header/header.css]: it's better to use rem values, not px.

File: ReviewSite/index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    <link rel="stylesheet" href="review/__title/review__title.css">
+    <link rel="stylesheet" href="review/__label/review__label.css">
+    <link rel="stylesheet" href="review-list/review-list.css">
+    <link rel="stylesheet" href="review/__description/__description.css">
+    <link rel="stylesheet" href="review/__image/review__image.css">
+</head>
+<body class="body">
+<header class="header">
+    <label class="header__title">Review Control</label>
+    <button class="header__button">Log out</button>
```
Comment (katetushkan) [inline ReviewSite/index.html]: it should be <a>, cuz it will redirect u to another page

File: ReviewSite/index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    <button class="header__button">Your account</button>
```
Comment (katetushkan) [inline ReviewSite/index.html]: same here.

File: ReviewSite/index.html [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+        <option>Film</option>
+        <option>Game</option>
+    </select>
+    <input class="search-panel__input" placeholder="Tag"/>
+    <input class="search-panel__input" placeholder="Rate"/>
+    <button class="search-panel__button">Go</button>
+</aside>
+<article class="review-list">
+    <article class="review">
+        <h5 class="review__title">ReviewTitle</h5>
```
Comment (katetushkan) [inline ReviewSite/index.html]: u can't use <h5> if u haven't use h4, h3, h2, h1 before

File: ReviewSite/review-list/review-list.css [hunk 7]
```diff
@@ -0,0 +1,10 @@
+.review-list {
+    width: calc(100% - 175px);
```
Comment (katetushkan) [inline ReviewSite/review-list/review-list.css]: hmmm? why did u do that?

File: ReviewSite/review-list/review-list.css [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+    float: right;
```
Comment (katetushkan) [inline ReviewSite/review-list/review-list.css]: no floats

File: ReviewSite/search-panel/__button/search-panel__button.css [hunk 9]
```diff
@@ -0,0 +1,6 @@
+.search-panel__button {
+    border: 1px solid black;
+    margin-top: 10px;
+    background-color: #1492AE;
+    width: 4em;
```
Comment (katetushkan) [inline ReviewSite/search-panel/__button/search-panel__button.css]: why u use em here?

File: ReviewSite/search-panel/search-panel.css [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+    padding: 20px;
+    margin: 10px;
+    text-align: center;
+    float: left;
+}
+@media (max-width: 500px) {
+    .search-panel {
+        float: unset;
+    }
+}
```
Comment (katetushkan) [inline ReviewSite/search-panel/search-panel.css:8]: Screenshot 2022-03-14 at 11 44 07
It doesn't look like ur mockup(

**Review by katetushkan — CHANGES_REQUESTED** [review]

File: ReviewSite/review/__description/review__description.css [hunk 11]
```diff
@@ -1,6 +1,6 @@
 .review__description {
-    margin: 10px;
-    width: 40%;
+    margin: 0.7rem;
+    width: 45vw;
```
Comment (katetushkan) [inline ReviewSite/review/__description/review__description.css]: don't take values from viewport, t can be tricky, if it's container it should be depending on a content inside, if no set it via px or rem

**Review by katetushkan — APPROVED** [review]
Rn It's an 8. If u wanna more, u need to add focus and active styles to ur button and link. Let me know if u wanna upgrade.

**Comment by Irina996** [comment]
I will upgrade

**Comment by Irina996** [comment]
please select the functions that I need to do so that I can start working on lab 4

**Comment by katetushkan** [comment]
> please select the functions that I need to do so that I can start working on lab 4

2) Search and filter reviews by title/tags/group/rate<br/>
 3) Like review<br/>
 4) Comment review<br/>


9 is urs. for the 3rd work.

**Review by katetushkan — COMMENTED** [review]
.

**Pull request merged**